public/sass/layouts/team_search_page.scss

```diff
@@ -2,6 +2,11 @@
   margin-top: 20px;
 }
 
+.create-team-btn {
+  margin-top: 2.0em;
+  margin-bottom: .71428571em;
```

Comment:
@Insti, The same `margin-bottom` value is used for `Search for public teams` H3 text.